include/z64player.h

```diff
@@ -713,10 +723,10 @@ typedef struct Player {
     /* 0x087C */ s16        unk_87C;
     /* 0x087E */ s16        unk_87E;
     /* 0x0880 */ f32        unk_880;
-    /* 0x0884 */ f32        wallHeight; // height used to determine whether link can climb or grab a ledge at the top
-    /* 0x0888 */ f32        wallDistance; // distance to the colliding wall plane
-    /* 0x088C */ u8         unk_88C;
-    /* 0x088D */ u8         unk_88D;
+    /* 0x0884 */ f32        yDistToLedge; // y distance to ground above an interact wall. LEDGE_DIST_MAX if no ground if found
+    /* 0x0888 */ f32        distToInteractWall; // xyz distance to the interact wall
```

Comment:
You've adjusted terminology and changed `InteractWall` -> `TouchedWall` in other places, should this also be changed to touched wall?

Reply:
They are two different things. The touched wall doesnt necessarily have to be the interact wall. They are the same in most cases but they can be different

Reply:
To try an elaborate, the touched wall is simply the wall that UpdateBgCheckInfo says youre touching. player does his own logic after the fact to flag that he is interacting with a wall after meeting other conditions. The cases where i changed interact -> touch is because its properties of the general touched wall, not necessarily the interact wall